docs/known-issues.md

```diff
@@ -20,7 +20,7 @@ Please check the [open](https://github.com/desktop/desktop/labels/bug) and [clos
 
 ### 'The username or passphrase you entered is not correct' error after signing into account - [#3263](https://github.com/desktop/desktop/issues/3263)
 
-This seems to be caused by the Keychain being in an invalid state, affecting applications that try to use the keychain to store or retrieve credentials. Seems to be specific to macOS High Sierra (10.13).
+This seems to be caused by the Keychain being in an invalid state, affecting applications that try to use the keychain to store or retrieve credentials. Seems to be specific to macOS High Sierra (10.13) to macOS Mojave 10.14.5 (18F132).
```

Comment:
```suggestion
This seems to be caused by the Keychain being in an invalid state, affecting applications that try to use the keychain to store or retrieve credentials. This has been reported from macOS High Sierra (10.13) to macOS Mojave 10.14.5 (18F132).
```

Reply:
Thanks 
yes i have experimented both  ...  on and on mojave just recently when i post this ... so since high sierra to mojave is true ... and the process to solve stay the same lock and unlock ....

Reply:
@jipexu Great, thanks. If you can go ahead and commit the suggested change I made we can get this merged in.

Reply:
@steveward sorry i dont know what i must do now .... ?

Reply:
@jipexu if you are fine with the wording change proposed by @steveward, the "Commit suggestion" button above will apply this change to the PR

<img width="478" src="https://user-images.githubusercontent.com/359239/61972886-10f7da00-afb9-11e9-8db3-2965789e3765.png">

Then I think this will be good to merge